docs/core_docs/docs/integrations/tools/stagehand.mdx

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_label: Stagehand AI Web Automation Toolkit
+hide_table_of_contents: true
+---
+
+import CodeBlock from "@theme/CodeBlock";
+import Example from "@examples/agents/stagehand_ai_web_browser.ts";
+
+# Stagehand Toolkit
+
+The Stagehand Toolkit equips your AI agent with the following capabilities:
+
+- **navigate()**: Navigate to a specific URL
+- **act()**: Perform browser automation actions like clicking, typing, and navigation
+- **extract()**: Extract structured data from web pages using Zod schemas
+- **observe()**: Get a list of possible actions and elements on the current page
+
+## Setup
+
+1. Install the required packages:
+
+```bash
+npm install @browserbasehq/stagehand zod
+```
+
+2. Create a Stagehand Instance
+   If you plan to run the browser locally, you'll also need to install Playwright's browser dependencies.
+
+```bash
+npm exec playwright install
```

Comment:
Should this be `npx` instead?

https://playwright.dev/docs/intro#updating-playwright

Reply:
@jacoblee93 updated!